tests/crypto/prj.conf

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# TEST
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=65536
+CONFIG_MAIN_STACK_SIZE=8096
```

Comment:
65k ztest_stacksize could be an issue for low-cost devices. Is this necessary?

8k is also pushing it.

Reply:
Sizes reduced considerably in 7aee7a9